.github/workflows/build.yml

```diff
@@ -36,6 +42,9 @@ jobs:
               TAG=$(TZ=UTC date +%Y%m%d)
           fi
           echo TAG=$TAG >> $GITHUB_OUTPUT
+          echo COMMIT_DATE_UNIX=$(git log -1 --pretty=%ct $GITHUB_SHA) >> $GITHUB_OUTPUT
+          echo BUILD_DATE=$(date -u '+%FT%TZ') >> $GITHUB_OUTPUT
+          echo BUILD_DATE_UNIX=$(date -d "${BUILD_DATE}" +%s) >> $GITHUB_OUTPUT
```

Comment:
Btw, I just realized, this doesn't work as intended. At this point, `${BUILD_DATE}` is still not set - we've only written it into `$GITHUB_OUTPUT`, which is a plain file that gets added to the environment starting with the next step in the job - but does not take effect immediately within the same script. So effectively, this executes `date -d "" +%s`, which produces a timestamp corresponding to the start of the current day, it seems.

Reply:
See https://github.com/mstorsjo/llvm-mingw/commit/fix-build-date for a fix for this - if this seems reasonable to you, I'll push that to master.

Reply:
Good catch, fix seems sensible

Reply:
Thanks, I'll push that then.

The failure mode here, in producing a timestamp which is quite near to the intended one, but still not quite the same, was quite surprising too :-)